src/main/webapp/app/shared/login/login.component.ts

```diff
@@ -34,7 +50,9 @@ export class LoginModalComponent implements AfterViewInit {
       username: '',
       password: '',
     });
-    this.activeModal.dismiss('cancel');
+    if (this.activeModal) {
```

Comment:
ich denke, die ganzen `if`s könnten mit der Syntax `this.activeModal?.dismiss('cancel');` verhindert werden.